test/unit/views/payment-method-view.js

```diff
@@ -160,7 +160,7 @@ describe('PaymentMethodView', () => {
       expect(config.model.confirmPaymentMethodDeletion).toBeCalledWith(config.paymentMethod);
     });
 
-    test('calls model.changeActivePaymentMethod when selection occurs when not in edit mode', () => {
+    test.skip('calls model.changeActivePaymentMethod when selection occurs when not in edit mode', () => {
```

Comment:
I'm not sure what this test needs to accomplish

Reply:
It ensures that the active payment method is updated (rather than deleted, like in the other test) when selected and not in the "edit payment method" view.